sql-plugins/src/main/java/io/cdap/pipeline/sql/plugins/bigquery/BigQueryUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.pipeline.sql.plugins.bigquery;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Table;
+import com.google.cloud.bigquery.TableId;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * BigQuery utility functions class.
```

Comment:
is this used in multiple places? In general, utility classes should be avoided if possible, as they tend to lead to unclean abstractions as more and more stuff gets added to them.

Reply:
Yeah I originally anticipated copying the functionality in multiple places then realized that it's only really used in the source. I'll go ahead and remove it.